mbta_finder.py

```diff
@@ -9,49 +12,56 @@
 
 from urllib.request import urlopen
 import json
-
-
-# Useful URLs (you need to add the appropriate parameters for your requests)
-GMAPS_BASE_URL = "https://maps.googleapis.com/maps/api/geocode/json"
-MBTA_BASE_URL = "http://realtime.mbta.com/developer/api/v2/stopsbylocation"
-MBTA_DEMO_API_KEY = "wX9NwuHnZU2ToO7GmGR9uw"
-
-
-# A little bit of scaffolding if you want to use it
-
-def get_json(url):
-    """
-    Given a properly formatted URL for a JSON web API request, return
-    a Python JSON object containing the response to that request.
-    """
-    pass
-
-
-def get_lat_long(place_name):
-    """
-    Given a place name or address, return a (latitude, longitude) tuple
-    with the coordinates of the given place.
-
-    See https://developers.google.com/maps/documentation/geocoding/
-    for Google Maps Geocode API URL formatting requirements.
-    """
-    pass
-
+from pprint import pprint
+
+
+def get_url(place_name):
+	GMAPS_BASE_URL = "https://maps.googleapis.com/maps/api/geocode/json"
+	place_name= place_name.replace(" ", "+")
```

Comment:
The urllib module has a function called urlencode that makes passing these parameters easier so you don't have to manually do string formatting.